src/cephadm/cephadm

```diff
@@ -2008,7 +2008,8 @@ def command_bootstrap():
         cli(['dashboard', 'ac-user-create',
              args.initial_dashboard_user, password,
              'administrator',
-             '--force-password'])
+             '--force-password',
+             '' if args.dashboard_password_noupdate else '--pwd-update-required'])
```

Comment:
I think in we don't want to pass an argument that is an empty string if this is false... maybe something like
```
cmd = [ ... ]
if not args.dashboard_password_noupdate:
    cmd.append('--pwd-update-required')
cli(cmd)
```

Reply:
Why not use a list for all command parameters and append `--pwd-update-required` if necessary? E.g. https://github.com/ceph/ceph/blob/master/qa/tasks/mgr/dashboard/helper.py#L57

Reply:
:+1: